src/ferc_xbrl_extractor/cli.py

```diff
@@ -183,22 +183,32 @@ def main():
         # Get most recent taxonomy for specified form number
         taxonomy = TAXONOMY_MAP[args.form_number]
 
-    instances = get_instances(
-        Path(args.instance_path),
-    )
-
-    xbrl.extract(
-        instances,
-        engine,
-        taxonomy,
-        args.form_number,
+    tables = xbrl.get_fact_tables(
```

Comment:
Pulling taxonomy generation out of the main `extract` function makes the interface simpler, but also lets us not double-generate the taxonomies.